src/github_runner_image_builder/openstack_builder.py

```diff
@@ -197,14 +198,15 @@ class CloudConfig:
         flavor: The OpenStack flavor to launch builder VMs on.
         network: The OpenStack network to launch the builder VMs on.
         proxy: The proxy to enable on builder VMs.
-        upload_cloud_name: The OpenStack cloud name to upload the snapshot to. (Default same cloud)
+        upload_cloud_names: The OpenStack cloud names to upload the snapshot to. (Defaults to \
```

Comment:
Is the `\` line break intended here?

Reply:
Yup, the line was too long! Thanks for the review!